opencaesar/oml-vision
# Feat/49-custom-deletion-wizards

## Checklist before submitting a merge request


## Description of contribution

Adds the ability for custom deletion wizards that opens on a command.
closes #49 

## Testing performed




## How to Test Expected functionality changes

1. Go to https://github.com/UTNAK/kepler16b-using-imce-vocabulary
2. Switch to `main` branch
3. Run `git pull`
4. From the command line run `./gradlew load` or from the `Setup Tasks` webview
5. Wait for that command to finish
6. Open the `decompositions-autogen` webview
7. Right Click on the `orbiter-propulsion-subsystem` node
8. Click `Delete Connection`
9. Verify that the `decompositions-autogen` webview is refreshed with the `orbiter-propulsion-subsystem` node removed
10. From the command line run `./gradlew save` or from the `Setup Tasks` webview
11. Verify that there are deletions in the `components.oml` and `masses.oml` files
12. Revert these changes after verifying if you don't want to commit and push these deletions

## Additional context

[If needed, you may add additional context]
